contrib/check-config.sh

```diff
@@ -157,6 +157,14 @@ echo 'Generally Necessary:'
 echo -n '- '
 if [ "$(stat -f -c %t /sys/fs/cgroup 2> /dev/null)" = '63677270' ]; then
 	echo "$(wrap_good 'cgroup hierarchy' 'cgroupv2')"
+	cgroupv2_controller_loc="/sys/fs/cgroup/cgroup.controllers"
+	# Prints available cgroupv2 controllers if the file exists
+	if test -f "$cgroupv2_controller_loc"; then
+		mapfile -t cgroupv2_controllers < $cgroupv2_controller_loc
+		wrap_good "- cgroupv2 controllers" "${cgroupv2_controllers[*]}"
```

Comment:
we should confirm that the file contains the following controllers:
- cpu
- cpuset
- memory
- io
- pids

Reply:
I'll update it to show if the file has `cpu,cpuset,memory,io,pids` enabled or missing similar to other checks.

Reply:
![image](https://user-images.githubusercontent.com/6939749/105185090-2326ea00-5b56-11eb-8ec0-2997170f1a59.png)
Is this better?